src/AddTodoForm.jsx

```diff
@@ -1,21 +1,39 @@
-function AddTodoForm(props) {
+import React from "react";
+
+function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = React.useState("");
+
+  function handleTitleChange(event) {
+    let newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  }
+
   function handleAddTodo(event) {
     event.preventDefault();
-
-    let todoTitle = event.target.elements.title.value;
+    if(todoTitle==="") {
+      alert("You have to write a todo")
+      return
+    }
     console.log(todoTitle);
```

Comment:
Using console.log() statements is fine but is not preferred.

Reply:
I'll apply this changes in next PR